addons/point_of_sale/tools/posbox/configuration/upgrade.sh

```diff
@@ -48,84 +48,90 @@ create_partition () {
     fi
 
-    echo "end dowloading raspbian"
+    echo "end dowloading raspios"
 }
 
-copy_raspbian () {
+copy_raspios () {
     umount -v /boot
 
-    # mapper raspbian
-    PART_RASPBIAN_ROOT=$(fdisk -l | tail -n 1 | awk '{print $1}')
+    # mapper raspios
+    PART_RASPIOS_ROOT=$(fdisk -l | tail -n 1 | awk '{print $1}')
     PART_ODOO_ROOT=$(fdisk -l | tail -n 2 | awk 'NR==1 {print $1}')
     PART_BOOT=$(fdisk -l | tail -n 3 | awk 'NR==1 {print $1}')
-    RASPBIAN=$(echo *raspbian*.img)
-    LOOP_RASPBIAN=$(kpartx -avs "${RASPBIAN}")
-    LOOP_RASPBIAN_ROOT=$(echo "${LOOP_RASPBIAN}" | tail -n 1 | awk '{print $3}')
-    LOOP_RASPBIAN_ROOT="/dev/mapper/${LOOP_RASPBIAN_ROOT}"
-    LOOP_BOOT=$(echo "${LOOP_RASPBIAN}" | tail -n 2 | awk 'NR==1 {print $3}')
+    RASPIOS=$(echo *raspios*.img)
+    LOOP_RASPIOS=$(kpartx -avs "${RASPIOS}")
+    LOOP_RASPIOS_ROOT=$(echo "${LOOP_RASPIOS}" | tail -n 1 | awk '{print $3}')
+    LOOP_RASPIOS_ROOT="/dev/mapper/${LOOP_RASPIOS_ROOT}"
+    LOOP_BOOT=$(echo "${LOOP_RASPIOS}" | tail -n 2 | awk 'NR==1 {print $3}')
     LOOP_BOOT="/dev/mapper/${LOOP_BOOT}"
 
     mount -o remount,rw /
-    # copy raspbian
-    dd if="${LOOP_RASPBIAN_ROOT}" of="${PART_RASPBIAN_ROOT}" bs=4M status=progress
-    e2fsck -fv "${PART_RASPBIAN_ROOT}" # resize2fs requires clean fs
+    # copy raspios
+    dd if="${LOOP_RASPIOS_ROOT}" of="${PART_RASPIOS_ROOT}" bs=4M status=progress
+    e2fsck -fv "${PART_RASPIOS_ROOT}" # resize2fs requires clean fs
 
     # Modify startup
-    mkdir -v raspbian
-    mount -v "${PART_RASPBIAN_ROOT}" raspbian
-    resize2fs "${PART_RASPBIAN_ROOT}"
-    chroot raspbian/ /bin/bash -c "sudo apt-get -y install kpartx"
+    mkdir -v raspios
+    mount -v "${PART_RASPIOS_ROOT}" raspios
+    resize2fs "${PART_RASPIOS_ROOT}"
+    chroot raspios/ /bin/bash -c "sudo apt-get -y install kpartx"
     PATH_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-    cp -v "${PATH_DIR}"/upgrade.sh raspbian/home/pi/
-    NBR_LIGNE=$(sed -n -e '$=' raspbian/etc/rc.local)
-    sed -ie "${NBR_LIGNE}"'i\. /home/pi/upgrade.sh; copy_iot' raspbian/etc/rc.local
-    cp -v /etc/fstab raspbian/etc/fstab
-    sed -ie "s/$(echo ${PART_ODOO_ROOT} | sed -e 's/\//\\\//g')/$(echo ${PART_RASPBIAN_ROOT} | sed -e 's/\//\\\//g')/g" raspbian/etc/fstab
-    mkdir raspbian/home/pi/config
-    find /home/pi -maxdepth 1 -type f ! -name ".*" -exec cp {} raspbian/home/pi/config/ \;
+    cp -v "${PATH_DIR}"/upgrade.sh raspios/home/pi/
+    NBR_LIGNE=$(sed -n -e '$=' raspios/etc/rc.local)
+    sed -ie "${NBR_LIGNE}"'i\. /home/pi/upgrade.sh; copy_iot' raspios/etc/rc.local
+    cp -v /etc/fstab raspios/etc/fstab
+    sed -ie "s/$(echo ${PART_ODOO_ROOT} | sed -e 's/\//\\\//g')/$(echo ${PART_RASPIOS_ROOT} | sed -e 's/\//\\\//g')/g" raspios/etc/fstab
+    mkdir raspios/home/pi/config
+    find /home/pi -maxdepth 1 -type f ! -name ".*" -exec cp {} raspios/home/pi/config/ \;
 
     # download latest IoT Box image and check integrity
+<<<<<<< HEAD
```

Comment:
I thin you had an issue rebasing your code ;)

Reply:
lol